### PR TITLE
eslint: lint all the things

### PR DESCRIPTION
**Pull request by quisquous**

### CODE REVIEW

File: webpack/webpack.config.cjs [hunk 1]
```diff
@@ -1,5 +1,3 @@
-'use strict';
```
Comment (panicstevenson) [inline webpack/webpack.config.cjs:1]: Are these fine to just go away? I thought they wanted to be here, but I admittedly have no idea what strict mode does.

Comment (quisquous) [reply webpack/webpack.config.cjs:1]: No, I think you're correct.  I think cjs files all need use strict in them, but I cannot seem to convince eslint that these are commonjs modules.

Comment (quisquous) [reply webpack/webpack.config.cjs:1]: As per https://eslint.org/docs/rules/strict#rule-details, if `'sourceType': 'module'` is set, then the strict mode rule is ignored.

Comment (quisquous) [reply webpack/webpack.config.cjs]: OK found some override options!!

**Pull request merged**